packages/frontend/src/components/common/Filters/configs.ts

```diff
@@ -165,3 +169,19 @@ export const getFilterRuleLabel = (
         value: valuesText,
     };
 };
+
+export const getFilterRuleTables = (
+    filterRule: FilterRule | DashboardFilterRule,
+    field: FilterableField,
+): string[] => {
+    if (isDashboardFilterRule(filterRule) && filterRule.tileTargets) {
+        return Object.values(filterRule.tileTargets).reduce<string[]>(
+            (tables, tileTarget) => {
+                return uniq([...tables, capitalize(tileTarget.tableName)]);
```

Comment:
should I be using table label here instead of table name?

Reply:
Yes, the table name is only used as an ID.

Reply:
Let's not capitalize the label. The label was defined by the user or automatically generated via the table name using the `friendlyName()` function that capitalizes it.

Reply:
@ZeRego 👍 updated